sql_composer/sql_composer.py

```
@@ -0,0 +1,185 @@
+            table_name = self.table.name
+            col_names_fmted = ", ".join(col_names)
+        else:
+            table_name = f"{self.table.name} AS {alias}"
+            col_names_fmted = ", ".join([f"{alias}.{c_name}" for c_name in col_names])
+
+        stmt = f"""
+        SELECT 
+            {col_names_fmted}
+        FROM {table_name}
+        {self.translator.query_criteria_to_sql(query_criteria, columns_by_name)}
+        """
```

Ruff (0.14.7)

Possible SQL injection vector through string-based query construction

(S608)